src/pages/sol/axial_loading.astro

```diff
@@ -128,59 +130,64 @@ All internal forces can be obtained from equilibrium analysis only
 
 <p>
     <InlineEquation equation='\\delta_T' /> ,  <InlineEquation equation='\\varepsilon_T' />  present in addition to elastic  <InlineEquation equation='\\delta_E' /> ,  <InlineEquation equation='\\varepsilon_E' />  (from internal forces). Superposition (small strains):
- <DisplayEquation equation='\\varepsilon_{tot} = \\varepsilon_{E} + \\varepsilon_{T}\\' /> 
-  <DisplayEquation equation='\\delta_{tot} = \\delta_{E} + \\delta_{T}\\' /> 
+ <DisplayEquation equation='\\varepsilon_{tot} = \\varepsilon_{E} + \\varepsilon_{T}\\' title="Total strain" background="True" /> 
+  <DisplayEquation equation='\\delta_{tot} = \\delta_{E} + \\delta_{T}\\' title="Total displacement" background="True" /> 
 </p>
 
 <Image src='/Solid_Mechanics/Axial Loading-Figures/Temperature_NoLoad.png' width='4'>Temperature changes with no applied loads</Image>
 
  A rod rests freely on a smooth horizontal surface. Temperature of the rod is raised by <InlineEquation equation='\\Delta T' />. Rod elongates by an amount.
 
-<DisplayEquation equation='\\delta_{T} = \\alpha \\Delta T L\\' />
+<DisplayEquation equation='\\delta_{T} = \\alpha \\Delta T L\\' title="Displacement from temperature changes." background="True" />
 
  Linear coefficient of thermal expansion <InlineEquation equation='\\alpha' />,  <InlineEquation equation='[\\alpha] = \\frac{1}{K},\\frac{1}{°C},...' />. This deformation is associated with an average thermal strain:
 
-<DisplayEquation equation='\\varepsilon_{T} = \\frac{\\delta_T}{L} = \\alpha T\\' />
+<DisplayEquation equation='\\varepsilon_{T} = \\frac{\\delta_T}{L} = \\alpha T\\' title="Strain from temperature changes." background="True" />
 
 <Image src='/Solid_Mechanics/Axial Loading-Figures/Temperature_TwoPlates.png' width='1.5'>  
     Temperature changes with statically indeterminate beam
 </Image>
 
  Initially, rod of length <InlineEquation equation='L' /> is placed between two supports at a distance <InlineEquation equation='L' /> from each other. With no internal forces, there is no stress or strain.
 
-<DisplayEquation equation='R_{A} = R_{B} = 0\\' />
-<DisplayEquation equation='R_{A} = F\\' />
+<DisplayEquation equation='R_{A} = R_{B} = 0\\' title="Force balance" background="True" />
```

Comment:
Titles should end in periods for consistency. There are a couple of other cases of missing periods on titles below.

```suggestion
<DisplayEquation equation='R_{A} = R_{B} = 0\\' title="Force balance." background="True" />
```